### ISSUE TITLE
Merging policies for this repository

### ISSUE THREAD

**Issue by tom-tan** [common-workflow-language/cwl-v1.2#89]
Are there any merging policies for this repository? I checked [contrib.md](https://github.com/common-workflow-language/cwl-v1.2/blob/main/contrib.md) but I could not find descriptions of it. It would be nice if there is a document for it.

For example:
- When a PR includes new tests, should we confirm that cwltool passes the given tests?
  - Currently CI does not check it.
- Who can merge pull requests?
  - Note: this question is about the policy for this repository rather than the permissions in Github.
- Are any approvals needed before merging?
  - one approval, several approvals or approvals of specific members are needed?

**Comment by mr-c** [comment]
Thank you @tom-tan , this is a good question.

The high level concept for the policy is that "PRs should be accepted when it is thought that the changes would be approved by the vote of the CWL Leadership Team for the next release."

Scenarios that suggest approval:

Scenario A: a clarification of the text, no change in behavior
Scenario B: a new conformance test, no objection from major CWL vendors
Scenario C: Something more complicated, and consensus was eventually reached

Scenarios that do not suggest approval:
Situation X: adds new feature or breaks existing feature: this is not appropriate for the CWL-v1.2 repository.
Situation Y: lack of consensus, objection from existing CWL implementor

We could add CI against the CWL reference implementation as an informative check, but sometimes we might want to merge here first, update the schema copy in cwltool, and then merge a fix there.

**Comment by tom-tan** [comment]
Thank you for clarifying, @mr-c !
It would be nice if it is documented (may be CONTRIBUTING.md or something? contrib.md seems to be a list of main contributers).

**Comment by mr-c** [comment]
@tom-tan Yes, we should document this policy. A PR to do that would be positively received :-)

The policy is also not set in stone. I'm open to suggestions as to who can approve and how many approvals are needed.

Currently only one approval is needed. Merging is possible once there is one approval by anyone in the https://github.com/orgs/common-workflow-language/teams/people team. We might want to restrict that a bit..

**Comment by tetron** [comment]
https://github.com/common-workflow-language/governance/blob/main/releases.md#release-process is what the leadership team voted on prior to the 1.2 release.

Linking / copying / expanding that to make a contributing document would be a good idea.
